ffmpeg/plan.sh

```diff
@@ -1,28 +1,47 @@
 pkg_name=ffmpeg
 pkg_origin=core
-pkg_version=3.3.2
+pkg_version=4.1
 pkg_description="A complete, cross-platform solution to record, convert and \
 stream audio and video."
 pkg_upstream_url=https://ffmpeg.org/
 pkg_license=('LGPL-2.1+' 'GPL-2.0+')
 pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
 pkg_source=https://ffmpeg.org/releases/${pkg_name}-${pkg_version}.tar.gz
-pkg_shasum=84cf294b6b2115879f347ae74ab411876c4298e17d3a0db68a55338bb88fa594
+pkg_shasum=e7196ba28d7527daee917babdcbebb56b04c77e0b20593f2dd0bc11528a27066
 pkg_bin_dirs=(bin)
 pkg_include_dirs=(include)
 pkg_lib_dirs=(lib)
 pkg_build_deps=(
   core/diffutils
+  core/expat
+  core/fontconfig
```

Comment:
You've got some dependencies declared both in `pkg_build_deps` and `pkg_deps`. If they're needed in both, defining them in `pkg_deps` should be sufficient.